roles/rsyslog/roles/output_roles/elasticsearch/tasks/main.yaml

```diff
@@ -53,7 +53,7 @@
 # Deploy configuration files
 - name: Set Elasticsearch facts
   set_fact:
-    rsyslog_role_packages: "{{ rsyslog_elasticsearch_packages | flatten([]) }}"
+    rsyslog_role_packages: "{{ rsyslog_elasticsearch_packages }}"
```

Comment:
see above

Reply:
#87 (comment) is applied here, as well.

Reply:
Note: var name `rsyslog_role_packages` was renamed to `__rsyslog_packages` and the var is set with instead of `set_fact` but in `var:`.